index.html

```diff
@@ -166,11 +169,56 @@ <h4>Skills</h4>
                 <li>Codepen</li>
                 <li>GitHub</li>
                 <li>GitLAb</li>
-                <li>Terminal</li>     
+                <li>Terminal</li>
             </ul>
         </section>
-        
+
+        <section id="greetings2">
+            <h1 class="GE">Get started</h1>
+            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
+                rem excepturi.</p>
+
+                 <form class="form" action="#" target="" method="get" name="formDatosPersonales">
```

Comment:
- [ ] please ensure that your Form uses the action [formspree service](https://formspree.io/html).
eg action value should be similar to https://formspree.io/f/{form_id}
`method` value equals `POST`
check more [here](https://formspree.io/html/)